handbook/ops/bizops/amplitude.md

```diff
@@ -26,14 +26,45 @@ Anything not based directly on analyzing Sourcegraph Cloud events is in Looker.
+
+```
+JSON_EXTRACT(argument,
+  '$.code_search.results.results_count') AS results_count,
+JSON_EXTRACT(argument,
+  '$.code_search.results.alert') AS alert,
+CASE
+  WHEN name = 'ViewRepogroup:python' THEN 'python'
+  WHEN name = 'ViewRepogroup:golang' THEN 'golang'
+  WHEN name = 'ViewRepogroup:android' THEN 'android'
+  WHEN name = 'ViewRepogroup:kubernetes' THEN 'kubernetes'
+  WHEN name = 'ViewRepogroup:kubernetes' THEN 'kubernetes'
+  WHEN name = 'ViewRepogroup:kubernetes' THEN 'kubernetes'
+  WHEN name = 'ViewRepogroup:stanford' THEN 'stanford'
+  WHEN name = 'ViewRepogroup:cncf' THEN 'cncf'
+ELSE
+NULL
+END
+repogroup_name,
+```
+
+#### Adding user properties
+
+User properties are the attributes of individual users. The ```amplitude_user_characteristics``` table defines all user properties, and this table is joined in with every event that is passed to Amplitude. These user properties include every A/B test from ```ab_test_users```. See more in the A/B testing doc (coming soon). 
+
+If an additional user property is added to this table, it will only be applied to events and users going forward. User properties are sent to Amplitude along with events, so new properties will only be added when a user triggers an event that is sent to Amplitude. When adding a user property, please add it to the [data map](https://docs.google.com/spreadsheets/d/1wz958I67BKWWY0jKY3oXKhlrGZ9ucKmv0CM94K-5NVs/edit#gid=735397811)
+
+#### Adding events
 1. A product/engineering team member should let BizOps/DataOps know if there's an event they would like added to Amplitude. 
-2. Since Amplitude events are based on ```sourcegraph_analytics.amplitude_events_v3``` an [INSERT statement](https://console.cloud.google.com/bigquery?pli=1&project=telligentsourcegraph&ws=!1m14!1m4!1m3!1stelligentsourcegraph!2sbquxjob_3a38e2f8_179cb5027f7!3sUS!1m4!4m3!1stelligentsourcegraph!2sdotcom_events!3samplitude_events_v2!1m3!8m2!1s839055276916!2sed7433a9cf0646a8a7c186c907b9accb&jobFilter=%255B%257B_22k_22_3A_22User%2520email_22_2C_22t_22_3A10_2C_22v_22_3A_22_5C_22ericbm%2540sourcegraph.com_5C_22_22_2C_22s_22_3Atrue%257D%255D&sq=839055276916:ed7433a9cf0646a8a7c186c907b9accb) needs to be run to add or backfill events. A BizOps/DataOps team member will run this INSERT statement.
+2. Since Amplitude events are based on ```sourcegraph_analytics.amplitude_events_v2``` an [INSERT statement](https://console.cloud.google.com/bigquery?pli=1&project=telligentsourcegraph&ws=!1m14!1m4!1m3!1stelligentsourcegraph!2sbquxjob_3a38e2f8_179cb5027f7!3sUS!1m4!4m3!1stelligentsourcegraph!2sdotcom_events!3samplitude_events_v2!1m3!8m2!1s839055276916!2sed7433a9cf0646a8a7c186c907b9accb&jobFilter=%255B%257B_22k_22_3A_22User%2520email_22_2C_22t_22_3A10_2C_22v_22_3A_22_5C_22ericbm%2540sourcegraph.com_5C_22_22_2C_22s_22_3Atrue%257D%255D&sq=839055276916:ed7433a9cf0646a8a7c186c907b9accb) needs to be run to add or backfill events. A BizOps/DataOps team member will run this INSERT statement.
```

Comment:
```suggestion
2. Since Amplitude events are based on the ```sourcegraph_analytics.amplitude_events_v2``` table, an [INSERT statement](https://console.cloud.google.com/bigquery?pli=1&project=telligentsourcegraph&ws=!1m14!1m4!1m3!1stelligentsourcegraph!2sbquxjob_3a38e2f8_179cb5027f7!3sUS!1m4!4m3!1stelligentsourcegraph!2sdotcom_events!3samplitude_events_v2!1m3!8m2!1s839055276916!2sed7433a9cf0646a8a7c186c907b9accb&jobFilter=%255B%257B_22k_22_3A_22User%2520email_22_2C_22t_22_3A10_2C_22v_22_3A_22_5C_22ericbm%2540sourcegraph.com_5C_22_22_2C_22s_22_3Atrue%257D%255D&sq=839055276916:ed7433a9cf0646a8a7c186c907b9accb) needs to be run to add or backfill events into the table. When this is run, all prior occurrences of the new event will be added to the table. When the scheduled query to update `sourcegraph_analytics.amplitude_events_v2` is next run, all new occurrences of the event will be populated to the table. A BizOps/DataOps team member will run this INSERT statement.
```

Is there also another step here missing where a bizops team member needs to add the new event to the `sourcegraph_analytics.amplitude_events_v2` scheduled query? (The main query)